frontend/src/components/network-presets/network-presets.tsx

```diff
@@ -0,0 +1,184 @@
+import { useMemo, useState } from 'react'
+
+import { useGlobal } from '../../contexts/global/global-context'
+import { useImportNetwork } from '../../hooks/use-import-network'
+import type { NetworkPreset } from '../../lib/networks'
+import { Button } from '../button'
+import { ButtonUnstyled } from '../button-unstyled'
+import { Header } from '../header'
+
+const hasImportedTestNetworks = (
+  testPresets: NetworkPreset[],
+  networks: string[]
+) => {
+  return testPresets.reduce<boolean>((acc, preset) => {
+    return acc || !!networks.find(n => n === preset.name)
+  }, false)
+}
+
+const itemStyles = {
```

Comment:
question(non-blocking): What's the reason you are not using tailwind css?

Reply:
no idea, i haven't written the repo, just inherited it a few weeks ago :)